start-all.sh

```diff
@@ -681,7 +688,7 @@ if [ "$SKIP_ALERTMANAGER" = "1" ]; then
 	AM_ADDRESS="127.0.0.1:9093"
 else
 	echo "Wait for alert manager container to start"
-	AM_ADDRESS=$(./start-alertmanager.sh $ALERTMANAGER_PORT $ALERT_MANAGER_DIR -D "$DOCKER_PARAM" $LIMITS $VOLUMES $PARAMS $ALERTMANAGER_COMMAND $BIND_ADDRESS_CONFIG $ALERT_MANAGER_RULE_CONFIG)
+	AM_ADDRESS=$(./start-alertmanager.sh $ALERTMANAGER_PORT "$QUICK_STARTUP_CMD" $ALERT_MANAGER_DIR -D "$DOCKER_PARAM" $LIMITS $VOLUMES $PARAMS $ALERTMANAGER_COMMAND $BIND_ADDRESS_CONFIG $ALERT_MANAGER_RULE_CONFIG)
```

Comment:
Are you sure you wanted the quotes around `"$QUICK_STARTUP_CMD"`? It means if this variable is empty (as it is by default), and empty string parameter is passed to the script. Which maybe doesn't affect it, but it's strange nonetheless.

Reply:
It does not exist by default add the quote make sure that it will not fail the script